### PR TITLE
chore: Add an entry for zip_fix engine

### PR DESCRIPTION
**Pull request by eseidel**
This adds an entry to artifact_proxy for my in-progress fix to the engine:
https://github.com/shorebirdtech/engine/tree/zip_fix

### CODE REVIEW

File: packages/artifact_proxy/lib/config.dart [hunk 1]
```diff
@@ -43,8 +45,8 @@ class _EngineMapping3_7 extends EngineMapping {
 /// Flutter 3.7.10
 const flutter_3_7_10 = {
   // Attempt to fix https://github.com/shorebirdtech/shorebird/issues/235
-  'f68335f3f6afdb1595420b1c3b5a16c8da75a1cf': _EngineMapping3_7(
-    shorebirdStorageBucket: 'download-dev.shorebird.dev',
+  'adb70a20d4718b5ce60cdd99ad81d8de54afcb35': _EngineMapping3_7(
+    // shorebirdStorageBucket: 'download-dev.shorebird.dev',
```
Comment (felangel) [inline packages/artifact_proxy/lib/config.dart:47]: You want this using the prod bucket?

Comment (eseidel) [reply packages/artifact_proxy/lib/config.dart:47]: It's already pushed in prod:
https://console.cloud.google.com/storage/browser/download.shorebird.dev/shorebird/adb70a20d4718b5ce60cdd99ad81d8de54afcb35?pageState=(%22StorageObjectListTable%22:(%22f%22:%22%255B%255D%22))&project=code-push-prod&prefix=&forceOnObjectsSortingFiltering=false

But I guess I could push it again, this time to dev. 🤷

Comment (felangel) [reply packages/artifact_proxy/lib/config.dart:47]: Up to you just wanted to make sure this was intentional

**Review by felangel — APPROVED** [review]
I’m confused about the unused variable warning haha but I’ll take a closer look when I’m at my computer.

**Comment by eseidel** [comment]
> I’m confused about the unused variable warning haha but I’ll take a closer look when I’m at my computer.

That's working around https://github.com/dart-lang/sdk/issues/52016 🤣

**Pull request merged**